docs/content/about-iota/about-iota.mdx

```diff
@@ -71,9 +70,9 @@ IOTA EVM is the distinct Layer 2 EVM running on top of the [IOTA Mainnet](./toke
 
 #### IOTA EVM Testnet
 
-<NotAvailableOnTestnet/>
+IOTA EVM Testnet is the distinct Layer 2 EVM running on top of the IOTA Testnet network.
```

Comment:
Please clarify that it runs on top of the Stardust protocol network, i.e., not available on the IOTA Rebased Testnet